src/__tests__/views/editArticle.test.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import { MemoryRouter } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+import configureStore from 'redux-mock-store';
+import EditArticle from '../../views/editArticle/Index';
+import Header from '../../components/header/Header';
+import Create from '../../components/createForms/Create';
+
+
+const props = {
+  history: {
+    push: jest.fn(),
+    location: {
+      pathname: ''
+    }
+  },
+
+  editArticle: jest.fn(),
+  article: {},
+  user: {},
+};
+
+const article = {
+  id: 1,
+  title: 'test',
+  description: 'description',
+  images: ['image'],
+  body: 'body',
+  updatedAt: 'time',
+  slug: 'slug',
+};
+
+const currentUser = {
+  username: 'testUsername',
+  email: 'test@email.com',
+  name: 'testName',
+  role: 'admin',
+};
+
+const mockStore = configureStore([thunk]);
+
+const store = mockStore({
+  authReducer: {
+    currentUser,
+    isLoggedIn: true
+  },
+  articleReducer: {
+    article,
+  },
+  component: 'Write'
+});
+
+const mountWrapper = mount(
+  <Provider store={store}>
+    <MemoryRouter>
+      <EditArticle {...props} />
+    </MemoryRouter>
+  </Provider>
+);
+
+describe('Article editing page', () => {
```

Comment:
@chingsley are we testing the page or the component?

Reply:
We have components and we have page views. The Article editing page view is what I am referring to here.

Reply:
The components used are the same components used for creating articles. Which have been tested in @leksyib's branch.

Reply:
@chingsley I don't understand this. We'll talk about it at our meeting.

Reply:
I have changed the description of the test to a more appropriate term